lib/round.rb

```diff
@@ -0,0 +1,111 @@
+
+class Round 
+
+    attr_reader :deck, :turns, :num_correct
+    def initialize(deck)
+        @deck = deck
+        @turns = []
+        @num_correct = []
+ 
+end
+
+def current_card
+    @deck.cards[@turns.count]
+end
+
+def take_turn(guess)
+    new_turn = Turn.new(guess, current_card)
+    @turns << new_turn
+    if new_turn.correct?
+        @num_correct << new_turn
```

Comment:
Instead of putting the turns into this num_correct, we could increment the value of num_correct. For example, we could set `@num_correct = 0` in the initialize method, and then on this line, we can do `@num_correct += 1`